lib/lifecycles/changelog.js

```diff
@@ -5,6 +5,7 @@ const conventionalChangelog = require('conventional-changelog')
 const fs = require('fs')
 const runLifecycleScript = require('../run-lifecycle-script')
 const writeFile = require('../write-file')
+const START_OF_LAST_RELEASE_PATTERN = /(^##? (?!Change Log$)|<a name=)/m
```

Comment:
Note that I really don't know of a bullet-proof way to do this because the format of where the "old content" begins is really dependent on the templates being used. For example, Angular may use either one `#` or two `##` depending on what kind of version it is (patch, etc).